conf/.env

```diff
@@ -1,26 +1,21 @@
 # Folder where data should be saved
-DATA_FOLDER=__DATADIR__
-
-# The top level domain to serve from
-DOMAIN_NAME=__DOMAIN__
-
-# The subdomain to serve from
-SUBDOMAIN=__PATH__
-
-# DOMAIN_NAME and SUBDOMAIN combined decide where n8n will be reachable from
-# above example would result in: https://n8n.example.com
```

Comment:
I may be incorrect, so please tell me if I am wrong.

I have searched for these variables in the n8n documentation. The only occurrences I found was here:
https://docs.n8n.io/getting-started/installation/advanced/server-setup.html#_6-create-env-file

However, this page seems to relate to Docker-Compose installations.

These variables are not listed here however:
https://docs.n8n.io/reference/environment-variables.html#endpoints